envpool/make_test.py

```diff
@@ -126,7 +126,7 @@ def test_make_mujoco_gym(self) -> None:
         "Walker2d-v4",
       ]
     )
-
+  # make test with GL loading error
```

Comment:
Seems like the python side also needs to load GL. Now it pops out `ERROR: gladLoadGL error`.